### ISSUE TITLE
Building using php:5.6-fpm image

### ISSUE THREAD

**Issue by mlalbuquerque** [docker-library/php#137]
I'm using your php:5.6-fpm image, but when installing extensions, I had some troubles. First, the ZLIB ext: I had to change the file "config0.m4" name to "config.m4" to install correctly.

And now I'm having problems with spl. It's printing this error:

cp: cannot stat 'modules/*': No such file or directory
Makefile:86: recipe for target 'install-modules' failed
make: *** [install-modules] Error 1
What it could be?

**Comment by yosifkit** [comment]
I believe zlib is already compiled in (https://github.com/docker-library/php/blob/master/5.6/fpm/Dockerfile#L56).  Can you post the relevant part of your Dockerfile?

**Comment by tianon** [comment]
As I mentioned over in https://github.com/docker-library/php/issues/121#issuecomment-173733019, ZLIB is included by default now.

As for "spl", from https://secure.php.net/manual/en/spl.installation.php:

> This extension is available and compiled by default in PHP 5.0.0.

Additionally:
```console
$ docker run -it --rm php:5 php -r 'var_export(spl_classes());'
array (
  'AppendIterator' => 'AppendIterator',
  'ArrayIterator' => 'ArrayIterator',
  'ArrayObject' => 'ArrayObject',
  'BadFunctionCallException' => 'BadFunctionCallException',
  'BadMethodCallException' => 'BadMethodCallException',
  'CachingIterator' => 'CachingIterator',
  'CallbackFilterIterator' => 'CallbackFilterIterator',
  'Countable' => 'Countable',
  'DirectoryIterator' => 'DirectoryIterator',
  'DomainException' => 'DomainException',
  'EmptyIterator' => 'EmptyIterator',
  'FilesystemIterator' => 'FilesystemIterator',
  'FilterIterator' => 'FilterIterator',
  'GlobIterator' => 'GlobIterator',
  'InfiniteIterator' => 'InfiniteIterator',
  'InvalidArgumentException' => 'InvalidArgumentException',
  'IteratorIterator' => 'IteratorIterator',
  'LengthException' => 'LengthException',
  'LimitIterator' => 'LimitIterator',
  'LogicException' => 'LogicException',
  'MultipleIterator' => 'MultipleIterator',
  'NoRewindIterator' => 'NoRewindIterator',
  'OuterIterator' => 'OuterIterator',
  'OutOfBoundsException' => 'OutOfBoundsException',
  'OutOfRangeException' => 'OutOfRangeException',
  'OverflowException' => 'OverflowException',
  'ParentIterator' => 'ParentIterator',
  'RangeException' => 'RangeException',
  'RecursiveArrayIterator' => 'RecursiveArrayIterator',
  'RecursiveCachingIterator' => 'RecursiveCachingIterator',
  'RecursiveCallbackFilterIterator' => 'RecursiveCallbackFilterIterator',
  'RecursiveDirectoryIterator' => 'RecursiveDirectoryIterator',
  'RecursiveFilterIterator' => 'RecursiveFilterIterator',
  'RecursiveIterator' => 'RecursiveIterator',
  'RecursiveIteratorIterator' => 'RecursiveIteratorIterator',
  'RecursiveRegexIterator' => 'RecursiveRegexIterator',
  'RecursiveTreeIterator' => 'RecursiveTreeIterator',
  'RegexIterator' => 'RegexIterator',
  'RuntimeException' => 'RuntimeException',
  'SeekableIterator' => 'SeekableIterator',
  'SplDoublyLinkedList' => 'SplDoublyLinkedList',
  'SplFileInfo' => 'SplFileInfo',
  'SplFileObject' => 'SplFileObject',
  'SplFixedArray' => 'SplFixedArray',
  'SplHeap' => 'SplHeap',
  'SplMinHeap' => 'SplMinHeap',
  'SplMaxHeap' => 'SplMaxHeap',
  'SplObjectStorage' => 'SplObjectStorage',
  'SplObserver' => 'SplObserver',
  'SplPriorityQueue' => 'SplPriorityQueue',
  'SplQueue' => 'SplQueue',
  'SplStack' => 'SplStack',
  'SplSubject' => 'SplSubject',
  'SplTempFileObject' => 'SplTempFileObject',
  'UnderflowException' => 'UnderflowException',
  'UnexpectedValueException' => 'UnexpectedValueException',
)
```
